src/js/edu-benefits/1995/config/form.js

```diff
@@ -0,0 +1,66 @@
+import { fullName, ssn } from '../../../common/schemaform/definitions';
+import { validateSSN } from '../../../common/schemaform/validation';
+import IntroductionPage from '../components/IntroductionPage';
+
+const formConfig = {
+  urlPrefix: '/1995/',
+  introduction: IntroductionPage,
+  confirmation: null,
+  chapters: {
+    veteranInformation: {
+      title: 'Veteran Information',
+      pages: {
+        veteranInformation: {
+          path: 'applicant-information',
+          initialData: {},
```

Comment:
If you want to set an initial value for any of the fields in the form. We don't need to initialize everything with empty strings like we do now in veteran.js, but we do sometimes set dates to the current date initially, for example.